src/assets/styles/theme.scss

```diff
@@ -61,4 +61,16 @@ $BCgovFontColorInverted: #ffffff;
 $app-blue: #1669bb; // same as the Vuetify theme primary
 $app-dk-blue: #38598a; // same as the Vuetify theme appDkBlue
 $app-red: #d3272c; // same as the Vuetify theme error
-$app-green: #1a9031, // same as the Vuetify theme success
+$app-green: #1a9031; // same as the Vuetify theme success
+
+// Sizes for Fonts
+$px-11: 0.6875rem;
+$px-12: 0.7500rem;
+$px-13: 0.8125rem;
+$px-14: 0.8750rem;
+$px-15: 0.9375rem;
+$px-16: 1.0000rem;
+
+// Sizes for Icons
+$px-22: 1.375rem;
+$px-25: 1.5625rem;
```

Comment:
Called these out explicitly as icons although we could add to the above list. 
Just can't recall the last time we have font this large outside of headers.

Good with either or

Reply:
I'd code the icons with their directly (ie, in rem) and reserve these variables for fonts only.